cipher/a1z26/a1z26.go

```diff
@@ -0,0 +1,76 @@
+// a1z26.go
+// description: Implementation of the a1z26 cipher
+// details:
+// The A1Z26 cipher is a simple substitution cipher where each letter is
+// replaced by the number of the order they're in. For example, A corresponds to
+// 1, B = 2, C = 3, etc..
+//
+// author(s): [Focusucof](https://github.com/Focusucof)
+// see a1z26_test.go for tests
+
+// ref: https://www.dcode.fr/letter-number-cipher
+package a1z26
+
+import (
+	"strconv"
+	"strings"
+	"regexp"
+)
+
+var encryptionMap = []string{"a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z"}
+
+// function to check if input is valid
+func isValidInput(input string) bool {
+	return regexp.MustCompile(`^[a-z A-Z]+$`).MatchString(input) // check if input contains only valid letters
+}
+
+// helper function to find the index of a given value in an array
```

Comment:
```suggestion
// indexOfLetter function to find the index of a given value in an array
```

Please use the name of the function as the first word in the documentation comment. Please read the commenting conventions of Go language.